2020-10-13 15:25:23 [scrapy.core.downloader.handlers.http11] WARNING: Got data loss in https://www.bilibili.com/ranking/origin/0/0/3?rt=V%2FymTlOu4ow%2Fy4xxNWPUZ%2FgyysuplfLt1nSFWW3JDFI%3D. If you want to process broken responses set the setting DOWNLOAD_FAIL_ON_DATALOSS = False -- This message won't be shown in further requests
2020-10-13 15:25:23 [scrapy.core.downloader.handlers.http11] WARNING: Got data loss in https://www.bilibili.com/ranking/all/0/0/3?rt=V%2FymTlOu4ow%2Fy4xxNWPUZ%2FgyysuplfLtc7T5%2FyJ7SDo%3D. If you want to process broken responses set the setting DOWNLOAD_FAIL_ON_DATALOSS = False -- This message won't be shown in further requests
2020-10-13 17:35:44 [scrapy.core.engine] ERROR: Error while obtaining start requests
Traceback (most recent call last):
  File "E:\Users\xcl\anaconda3\lib\site-packages\scrapy\core\engine.py", line 129, in _next_request
    request = next(slot.start_requests)
  File "E:\PyDev\bilipy\bilipy\spiders\bilibili.py", line 22, in start_requests
    os.remove('txts\\*.txt')
OSError: [WinError 123] 文件名、目录名或卷标语法不正确。: 'txts\\*.txt'
2020-10-13 17:36:29 [scrapy.core.engine] ERROR: Error while obtaining start requests
Traceback (most recent call last):
  File "E:\Users\xcl\anaconda3\lib\site-packages\scrapy\core\engine.py", line 129, in _next_request
    request = next(slot.start_requests)
  File "E:\PyDev\bilipy\bilipy\spiders\bilibili.py", line 22, in start_requests
    os.remove('txts\\*.txt')
OSError: [WinError 123] 文件名、目录名或卷标语法不正确。: 'txts\\*.txt'
2020-10-13 17:37:37 [scrapy.core.engine] ERROR: Error while obtaining start requests
Traceback (most recent call last):
  File "E:\Users\xcl\anaconda3\lib\site-packages\scrapy\core\engine.py", line 129, in _next_request
    request = next(slot.start_requests)
  File "E:\PyDev\bilipy\bilipy\spiders\bilibili.py", line 22, in start_requests
    os.remove('E:\PyDev\\bilipy\\bilipy\\txts\\*.txt')
OSError: [WinError 123] 文件名、目录名或卷标语法不正确。: 'E:\\PyDev\\bilipy\\bilipy\\txts\\*.txt'
2020-10-13 17:39:42 [scrapy.core.engine] ERROR: Error while obtaining start requests
Traceback (most recent call last):
  File "E:\Users\xcl\anaconda3\lib\site-packages\scrapy\core\engine.py", line 129, in _next_request
    request = next(slot.start_requests)
  File "E:\PyDev\bilipy\bilipy\spiders\bilibili.py", line 22, in start_requests
    os.remove('E:/PyDev/bilipy/bilipy/txts/*.txt')
OSError: [WinError 123] 文件名、目录名或卷标语法不正确。: 'E:/PyDev/bilipy/bilipy/txts/*.txt'
2020-10-13 17:40:28 [scrapy.core.engine] ERROR: Error while obtaining start requests
Traceback (most recent call last):
  File "E:\Users\xcl\anaconda3\lib\site-packages\scrapy\core\engine.py", line 129, in _next_request
    request = next(slot.start_requests)
  File "E:\PyDev\bilipy\bilipy\spiders\bilibili.py", line 22, in start_requests
    os.unlink('E:/PyDev/bilipy/bilipy/txts/*.txt')
OSError: [WinError 123] 文件名、目录名或卷标语法不正确。: 'E:/PyDev/bilipy/bilipy/txts/*.txt'
2020-10-13 17:43:34 [scrapy.core.engine] ERROR: Error while obtaining start requests
Traceback (most recent call last):
  File "E:\Users\xcl\anaconda3\lib\site-packages\scrapy\core\engine.py", line 129, in _next_request
    request = next(slot.start_requests)
  File "E:\PyDev\bilipy\bilipy\spiders\bilibili.py", line 22, in start_requests
    os.unlink('E:/PyDev/bilipy/bilipy/txts/*.txt'.strip('\u202a'))
OSError: [WinError 123] 文件名、目录名或卷标语法不正确。: 'E:/PyDev/bilipy/bilipy/txts/*.txt'
2020-10-13 17:43:58 [scrapy.core.engine] ERROR: Error while obtaining start requests
Traceback (most recent call last):
  File "E:\Users\xcl\anaconda3\lib\site-packages\scrapy\core\engine.py", line 129, in _next_request
    request = next(slot.start_requests)
  File "E:\PyDev\bilipy\bilipy\spiders\bilibili.py", line 22, in start_requests
    os.unlink('E:/PyDev/bilipy/bilipy/txts/*.txt'.strip())
OSError: [WinError 123] 文件名、目录名或卷标语法不正确。: 'E:/PyDev/bilipy/bilipy/txts/*.txt'
2020-10-13 21:36:00 [scrapy.core.scraper] ERROR: Spider error processing <GET https://www.bilibili.com/ranking/rookie/0/0/3?rt=V%2FymTlOu4ow%2Fy4xxNWPUZ3MgvaJsTsYcgognZBhln94%3D> (referer: None)
Traceback (most recent call last):
  File "E:\python37\lib\site-packages\scrapy\utils\defer.py", line 120, in iter_errback
    yield next(it)
  File "E:\python37\lib\site-packages\scrapy\utils\python.py", line 353, in __next__
    return next(self.data)
  File "E:\python37\lib\site-packages\scrapy\utils\python.py", line 353, in __next__
    return next(self.data)
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 29, in process_spider_output
    for x in result:
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 340, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "F:\pydev\bilipy\bilipy\spiders\bilibili.py", line 53, in parse
    danmu = video.get_danmaku(bvid=bvid)
  File "E:\python37\lib\site-packages\bilibili_api\video.py", line 69, in get_danmaku
    info = get_video_info(aid=aid, bvid=bvid, verify=verify)
  File "E:\python37\lib\site-packages\bilibili_api\video.py", line 46, in get_video_info
    info = utils.get(url=api["url"], params=params, cookies=verify.get_cookies())
  File "E:\python37\lib\site-packages\bilibili_api\utils.py", line 400, in get
    resp = request("GET", url=url, params=params, cookies=cookies, headers=headers, **kwargs)
  File "E:\python37\lib\site-packages\bilibili_api\utils.py", line 377, in request
    raise exceptions.BilibiliException(con["code"], msg)
bilibili_api.exceptions.BilibiliException: 错误代码：-404, 信息：啥都木有
2020-10-13 21:41:17 [scrapy.core.scraper] ERROR: Spider error processing <GET https://www.bilibili.com/ranking/origin/0/0/3?rt=V%2FymTlOu4ow%2Fy4xxNWPUZ3MgvaJsTsYcaLr6%2FAuS1Bg%3D> (referer: None)
Traceback (most recent call last):
  File "E:\python37\lib\site-packages\scrapy\utils\defer.py", line 120, in iter_errback
    yield next(it)
  File "E:\python37\lib\site-packages\scrapy\utils\python.py", line 353, in __next__
    return next(self.data)
  File "E:\python37\lib\site-packages\scrapy\utils\python.py", line 353, in __next__
    return next(self.data)
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 29, in process_spider_output
    for x in result:
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 340, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "F:\pydev\bilipy\bilipy\spiders\bilibili.py", line 53, in parse
    danmu = video.get_danmaku(bvid=bvid)
  File "E:\python37\lib\site-packages\bilibili_api\video.py", line 69, in get_danmaku
    info = get_video_info(aid=aid, bvid=bvid, verify=verify)
  File "E:\python37\lib\site-packages\bilibili_api\video.py", line 46, in get_video_info
    info = utils.get(url=api["url"], params=params, cookies=verify.get_cookies())
  File "E:\python37\lib\site-packages\bilibili_api\utils.py", line 400, in get
    resp = request("GET", url=url, params=params, cookies=cookies, headers=headers, **kwargs)
  File "E:\python37\lib\site-packages\bilibili_api\utils.py", line 377, in request
    raise exceptions.BilibiliException(con["code"], msg)
bilibili_api.exceptions.BilibiliException: 错误代码：-404, 信息：啥都木有
2020-10-13 21:41:34 [scrapy.core.scraper] ERROR: Spider error processing <GET https://www.bilibili.com/ranking/all/0/0/3?rt=V%2FymTlOu4ow%2Fy4xxNWPUZ3MgvaJsTsYcRIt302gv%2Fbo%3D> (referer: None)
Traceback (most recent call last):
  File "E:\python37\lib\site-packages\scrapy\utils\defer.py", line 120, in iter_errback
    yield next(it)
  File "E:\python37\lib\site-packages\scrapy\utils\python.py", line 353, in __next__
    return next(self.data)
  File "E:\python37\lib\site-packages\scrapy\utils\python.py", line 353, in __next__
    return next(self.data)
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\offsite.py", line 29, in process_spider_output
    for x in result:
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\referer.py", line 340, in <genexpr>
    return (_set_referer(r) for r in result or ())
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\urllength.py", line 37, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "E:\python37\lib\site-packages\scrapy\spidermiddlewares\depth.py", line 58, in <genexpr>
    return (r for r in result or () if _filter(r))
  File "E:\python37\lib\site-packages\scrapy\core\spidermw.py", line 62, in _evaluate_iterable
    for r in iterable:
  File "F:\pydev\bilipy\bilipy\spiders\bilibili.py", line 53, in parse
    danmu = video.get_danmaku(bvid=bvid)
  File "E:\python37\lib\site-packages\bilibili_api\video.py", line 69, in get_danmaku
    info = get_video_info(aid=aid, bvid=bvid, verify=verify)
  File "E:\python37\lib\site-packages\bilibili_api\video.py", line 46, in get_video_info
    info = utils.get(url=api["url"], params=params, cookies=verify.get_cookies())
  File "E:\python37\lib\site-packages\bilibili_api\utils.py", line 400, in get
    resp = request("GET", url=url, params=params, cookies=cookies, headers=headers, **kwargs)
  File "E:\python37\lib\site-packages\bilibili_api\utils.py", line 377, in request
    raise exceptions.BilibiliException(con["code"], msg)
bilibili_api.exceptions.BilibiliException: 错误代码：-404, 信息：啥都木有
